x-pack/plugins/maps/public/elasticsearch_geo_utils.test.js

```diff
@@ -59,6 +59,76 @@ describe('hitsToGeoJson', () => {
     });
   });
 
+  it('Should convert wkt shapes to geojson', () => {
```

Comment:
Can you add a test case for geohash geo_point?

Reply:
I see the geohash now in "Should handle point as string case". How about moving geohash test to `geoPointToGeometry` describe block since that is where the implemenation specific tests are. Then there is no need for `Should handle point as string case`.

The same with WKT test. That test should be under the `geoShapeToGeometry` describe block

Reply:
ok will do this